pkg/cmd/codespace/ssh.go

```diff
@@ -449,27 +412,39 @@ func getConfiguredPrivateKeys(
 		return nil, fmt.Errorf("could not load ssh configuration: %w", err)
 	}
 
-	var privateKeyPaths []string
-
 	userHomeDir, _ := os.UserHomeDir()
```

Comment:
I know this was already checked in, but should we be checking this error?

Reply:
We can, according to [the source](https://cs.opensource.google/go/go/+/refs/tags/go1.19.1:src/os/file.go;l=499) it can really only happen if the user home envvar (`$HOME`/`%userprofile%`) isn't set, which felt like something we didn't really need to worry about so much

Reply:
Added an error check